test/e2e/specs/test.spec.js

```diff
@@ -1,7 +1,8 @@
 /* eslint-disable ui-testing/no-disabled-tests */
 describe('Wallet App Test Cases', () => {
   context('Test commands', () => {
-    it(`should connect with Agoric Chain`, () => {
+    it(`should setup wallet and connect with Agoric Chain`, () => {
+      cy.setupWallet();
```

Comment:
we dont have the `SKIP_EXTENSION_SETUP` flag enabled anywhere, wont this duplicate the setup of the wallet?